styles/_slideshow.scss

```diff
@@ -0,0 +1,59 @@
+.slideshow {
+    margin: 0 auto;
+    position: relative;
+    overflow: hidden;
+
+    .action {
+        background-color: hsla(0deg, 10%, 70%, 0.2);
+        border: 1px solid #666666;
+        border-radius: 50%;
+        color: #666666;
+        cursor: pointer;
+        font-size: 32px;
+        height: 30px;
+        line-height: 22px;
+        position: absolute;
+        text-align: center;
+        top: 50%;
+        width: 30px;
+        z-index: 10;
+
+        &.left {
+            left: 10px;
+        }
+
+        &.right {
+            right: 10px;
+        }
+
+        &:hover {
+            background-color: hsla(0deg, 10%, 70%, 0.1);
+            border: 1px solid #cccccc;
+            color: #cccccc;
+        }
+    }
+
+    .slides {
+        display: flex;
+        overflow-x: auto;
+        scroll-snap-type: x mandatory;
+        scroll-behavior: smooth;
+        -webkit-overflow-scrolling: touch;
+    }
+
+    .slides > img {
+        align-items: center;
+        display: flex;
+        flex-shrink: 0;
+        font-size: 100px;
+        height: 100%;
+        justify-content: center;
+        margin-right: 50px;
+        scroll-snap-align: start;
+        position: relative;
+        transform-origin: center center;
+        transform: scale(1);
+        transition: transform 0.5s;
+        width: 100%;
+    }
```

Comment:
```suggestion

        > img {
            align-items: center;
            display: flex;
            flex-shrink: 0;
            font-size: 100px;
            height: 100%;
            justify-content: center;
            margin-right: 50px;
            scroll-snap-align: start;
            position: relative;
            transform-origin: center center;
            transform: scale(1);
            transition: transform 0.5s;
            width: 100%;
        }
    }
```

Reply:
Moved the img styles inside the parent def.